test/extensions/INTEL/SPV_INTEL_subgroup_requirements/named_sub_group_sizes.ll

```diff
@@ -19,12 +19,15 @@
 ; CHECK-SPIRV: ExecutionMode [[kernel]] 6446 0
 
 ; CHECK-LLVM: spir_kernel void @_ZTSZ4mainE7Kernel1() {{.*}} !intel_reqd_sub_group_size ![[MD:[0-9]+]]
-; CHECK-LLVM: ![[MD]] = !{i32 0}
+; CHECK-LLVM: ![[MD]] = !{i32 -1}
```

Comment:
Good point, I can see how that could be confusing. From my understanding, since LLVM does not have signed or unsigned integers types, when displaying integer literals, if a number is negative under two's complement in the a given integer type, then it'll it display the signed version of that integer, and not the unsigned version (e.g. see https://godbolt.org/z/xcjjPqvqf). So the value does remain to be 4294967295, but it's just displayed as -1.